lib/src/chart/line_chart/line_chart_renderer.dart

```diff
@@ -10,14 +10,24 @@ import 'package:flutter/material.dart';
 
 /// Low level LineChart Widget.
 class LineChartLeaf extends LeafRenderObjectWidget {
-  const LineChartLeaf({Key? key, required this.data, required this.targetData})
-      : super(key: key);
+  const LineChartLeaf({
+    Key? key,
+    required this.data,
+    required this.targetData,
+    this.longPressDuration,
+  }) : super(key: key);
 
   final LineChartData data, targetData;
+  final Duration? longPressDuration;
```

Comment:
We need to hold this variable in the `LineTouchData` class.

Reply:
@imaNNeoFighT 

Thnaks you for response. 
Fixed it. Moved the 'longPressDuration' option property to the 'LineTouchData' class as below.
https://github.com/imaNNeoFighT/fl_chart/pull/1128/commits/575fe536822850985159e59c98a9f26165bfa3e6

Reply:
Thank you. 
I think it's better to move it to our base class [FlTouchData](https://github.com/imaNNeoFighT/fl_chart/blob/0d8ecade338f0a7dc501f60c18b2f18c0d63516d/lib/src/chart/base/base_chart/base_chart_data.dart#L94). 
In this way, we can support it for all types of the charts (not only LineChart). And we don't need to pass it manually in our `RenderBaseChart` classes. Because `RenderBaseChart` already has access to the `touchData` which contains `longPressDuration`.
I'm really sorry about that.

Reply:
@imaNNeoFighT 

OK now, I moved `longPressDuration` property to `FlTouchData` base class. 
In addition, I added it to not only `LineTouchData` but also other charts(ex. pie_chart, bar_chart, etc) class.

>And we don't need to pass it manually in our RenderBaseChart classes. Because RenderBaseChart already has access to the touchData which contains longPressDuration.

Thanks for your advice. But, I have a question. how can we get `longPressDuration` property from `RenderBaseChart` class without manually argument?
Please show me to your expected code🙏

https://github.com/HTsuruo/fl_chart/blob/65763bf00472b7fb3d41505e205aee62a7596c33/lib/src/chart/base/base_chart/render_base_chart.dart#L72

Reply:
I reviewed them. Everything seems good. Thanks